lib/registrant.rb

```diff
@@ -0,0 +1,20 @@
+class Registrant 
+    attr_reader :name,
+                :age,
+                :license_data
+
+    def initialize(name, age, permit = false)
+        @name = name
+        @age = age
+        @permit = permit
+        @license_data = {:license=>false, :renewed=>false, :written=>false}
+    end
+
+    def permit?
+        @permit
+    end
+
+    def earn_permit
+        @permit = true
```

Comment:
Nice job here. You could also add what we call a guard clause like
```ruby
@permit = true if @age >= 15
```
This just makes sure that only eligible registrants can earn a permit.